### ISSUE TITLE
[docs/helm] Changelog is inconsistent, versioning is hard to follow

### ISSUE THREAD

**Issue by mjnagel** [grafana/loki#9120]
**Describe the bug**

I've been using the Loki helm chart (https://github.com/grafana/loki/tree/main/production/helm/loki) and recently discovered there was a new major version of the chart available (5.0.0). However in browsing [the changelog](https://github.com/grafana/loki/blob/main/production/helm/loki/CHANGELOG.md) I was unable to find a changelog entry for this version so it was hard to see what changes were included.

Diving in to the specific commits under the `production/helm/loki` file that were included in this new version I see:
- Docs/Typo Fixes: https://github.com/grafana/loki/commit/2cccc1bc6893c0b129bf2c81dbcd449ff837a53d
- PSP Change: https://github.com/grafana/loki/commit/c4d016fd53de7c4af0a4a6d60d191859d244417d (note the changelog entry was added under 4.10.0)
- DNS port rename: https://github.com/grafana/loki/commit/e2c4454a2d7deeec9996eb4c8a6303e44f4a996f (note a changelog addition above 4.10.0)
- Version bump to Loki 2.8.0 + chart bump: https://github.com/grafana/loki/commit/5113906c6723234c1441838c0d5896f28d15d3e3

It's not clear to me what the deciding factor was to make this a major release of the chart.

**To Reproduce**

This is primarily a "docs bug" so there are no real reproduction steps. The above links can be reviewed to observe the behavior of an end user trying to understand the update.

**Expected behavior**

The changelog should clearly lay out what changes were included in a given version. Another step that might be expected would be a link to the PR where the change was introduced (as indicated [here](https://github.com/grafana/loki/blob/main/production/helm/loki/CHANGELOG.md?plain=1#L12)).

It might also be valuable to have a clearly defined versioning schema that is adhered to, so that end users can understand what types of changes are included in a major, minor, or patch release. Based on the 5.0.0 update being a major release I'm not sure that semver is being adhered to here (I did not observe any breaking/incompatible API changes in the above commits).

**Environment:**
 - Infrastructure: Kubernetes
 - Deployment tool: helm

**Screenshots, Promtail config, or terminal output**

N/A

**Comment by JStickler** [comment]
Closing as Stale
As of March 16, 2026, the Loki Helm Chart has been forked to the new [Grafana-Community/helm-charts repo](https://github.com/grafana-community/helm-charts) and is now being maintained by the Grafana Community and Grafana Champions. If this issue affects Grafana Enterprise Logs (GEL), please leave a comment here.  Otherwise, if this is still an issue, please reopen your issue here [here](https://github.com/grafana-community/helm-charts/issues).
